NuGet.Config

```diff
@@ -29,5 +29,6 @@
     <add key="dotnet.myget.org roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
     <add key="myget.org vs-editor" value="https://myget.org/F/vs-editor/api/v3/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="devdiv.pkgs.visualstudio.com" value="https://devdiv.pkgs.visualstudio.com/_packaging/ApexTestFramework/nuget/v3/index.json" />
```

Comment:
The explosion of feeds is one of the reasons why our restore is so slow. We really need to find a way to consolidate the feeds at some point.

Reply:
APEX will definitely be published in myget not internally as definted here. Do you mean that we can decrease the number of nuget feeds?

---
In reply to: [162495545](https://github.com/dotnet/roslyn/pull/22201#discussion_r162495545) [](ancestors = 162495545)